lochness/redcap/process_piis.py

```diff
@@ -154,9 +155,10 @@ def process_pii_string(pii_string: str, process: str, subject_id: str) -> str:
         return get_shuffle_dict_for_type(letters, pii_string.upper())
 
     elif process == 'random_string':
+        system_random = random.SystemRandom()
```

Comment:
I see that line 95 is in a different function. Yet, don't we need any relation between the two `random.SystemRandom()` call?

Reply:
no, we don't need any relation between the two calls. (It's just another random function suggested by `Veracode` instead of `random.choice` module.)